Cargo.toml

```diff
@@ -68,15 +68,9 @@ version = "2.0"
 [dependencies.hex]
 version = "0.4"
 
-[dependencies.hyper]
-version = "0.14"
-features = [ "http1", "runtime", "server", "tcp" ]
-
-[dependencies.json-rpc-types]
-version = "1.0.2"
-
-[dependencies.jsonrpc-core]
-version = "18"
+[dependencies.jsonrpsee]
```

Comment:
Do we want to feature gate the rpc module?

Reply:
probably not during the testnet, but in the future, likely; I could add it as a default feature